lib/platform/bitbucket-server/index.ts

```diff
@@ -964,6 +964,7 @@ export function getPrBody(input: string) {
     .replace(/<\/?summary>/g, '**')
     .replace(/<\/?details>/g, '')
     .replace(new RegExp(`\n---\n\n.*?<!-- .*?-rebase -->.*?(\n|$)`), '')
+    .replace(new RegExp('<!--.*-->', 'g'), '')
```

Comment:
Shouldn't we be non greedy here `'<!--.*?-->'` ?

otherwise we would replace all:
```html
<!-- comment -->
Text
<!-- other comment -->
```

Reply:
Yeah, you're right. I will make sure that it works well with such scenarios as well.

Reply:
I've added the suggestion. if you like you can take it 😄

Reply:
It's a good suggestion. Happy to use it.

Reply:
It's a good suggestion. At least I can add